docs/conf.sh

```diff
@@ -13,6 +13,12 @@
 #this variable should be set somewhere other than here or it will not work
 YTFZF_CONFIG_DIR=$HOME/.config/ytfzf
 
+#the file to source for config
+#the best way to use this would either be to export it in your shell's rc or
+#to run ytfzf as such:
+    #YTFZF_CONFIG_FILE=path/to/config ytfzf ...
+YTFZF_CONFIG_FILE=$YTFZF_CONFIG_DIR/conf.sh
```

Comment:
This might be confusing, maybe keep it commented and tell them to add it their rc. Having it uncommented  may confuse them if they try editing it

Reply:
ok, im going to do the same with YTFZF_CONFIG_DIR